src/main.py

```diff
@@ -111,11 +113,22 @@ def metric_page():
             end_datetime_utc=endtime,
             model_name=model_name,
         )
+        # get metric value for mae with pvlive gsp sum truths for comparison
+        metric_values_mae_gsp_sum = get_metric_value(
+            session=session,
+            name=name_mae_gsp_sum,
+            gsp_id=0,
```

Comment:
could you try with out this gsp_id=0, and i think th esum of trusths does not have a gsp_id